lumeer-core/src/main/java/io/lumeer/core/facade/GroupFacade.java

```diff
@@ -41,33 +43,46 @@
    @Inject
    private OrganizationDao organizationDao;
 
-   public Group createGroup(String organizationId, Group group){
-      checkPermissions(organizationId);
+   public Group createGroup(Group group){
+      checkPermissions();
 
-      return groupDao.createGroup(organizationId, group);
+      return groupDao.createGroup(group);
    }
 
-   public Group updateGroup(String organizationId, String groupId, Group group){
-      checkPermissions(organizationId);
+   public Group updateGroup(String groupId, Group group){
+      checkPermissions();
 
       return groupDao.updateGroup(groupId, group);
    }
 
-   public void deleteGroup(String organizationId, String groupId){
-      checkPermissions(organizationId);
+   public void deleteGroup( String groupId){
+      checkPermissions();
 
-      userDao.deleteGroupFromUsers(organizationId, groupId);
       groupDao.deleteGroup(groupId);
+      deleteInfoAboutGroup(groupId);
```

Comment:
Isn't deleting the group in the line above this one going to remove all information about the group? I simply don't know what this method does unless I look into its implementation. And that's exactly what we have discussed with @marvec. If we name methods like this, we will definitely need comments in our code. But we can do better and avoid them. :)

I would suggest renaming it to `removeGroupFromUsers()` but feel free to find a better name if you wish.